test-scripts/local-is-deployment/DeploymentPatterns/pattern-2/run-scenarios.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+echo "This is a Dummy Scenario execution scripts"
```

Comment:
Why are we not calling the jmeter script? That logic may come in the next iteration, but in that case we have to remove the jmeter scripts from this PR. :)

Reply:
The tests do get executed in this case, with existing execution model. We can alter this when the new implementation comes into the picture.